 이상한-문자-만들기.js

```diff
@@ -0,0 +1,15 @@
+function solution(s) {
+
+
+    return s.split(' ').map(word => {
```

Comment:
불필요한 줄바꿈 지우기!
+ map함수 사용시 function element는 ()로 묶어주세요
```suggestion
function solution(s) {
    return s.split(' ').map((word) => {
```